rosdep/python.yaml

```diff
@@ -3087,6 +3087,14 @@ python-scipy:
     yakkety_python3: [python3-scipy]
     zesty: [python-scipy]
     zesty_python3: [python3-scipy]
+python-seaborn:
+  arch: [python-seaborn]
+  debian: [python-seaborn]
+  fedora:
+    pip:
+      packages: [seaborn]
+  gentoo: [dev-python/seaborn]
+  ubuntu: [python-seaborn]
```

Comment:
This package is not available on ubuntu trusty, we need to add a pip rule for trusty